services/main.go

```diff
@@ -0,0 +1 @@
+package main
```

Comment:
can have an implementation in the future when we have multiple game micro services and we want to compile it all in one binary :)